project2/Makefile

```diff
@@ -3,5 +3,10 @@ BOARD_TAG    = mega
 BOARD_SUB    = atmega2560
 ARDUINO_PORT = ${BASE_PORT}
 MONITOR_BAUDRATE = 38400
+MONITOR_CMD = miniterm.py --menu-char 27
```

Comment:
what char is 27?

Reply:
The other square bracket, I couldn't get miniterm to work for me on the normal chars